docs/explanations/qrao_explanations.rst

```diff
@@ -0,0 +1,368 @@
+Background on Quantum Random Access Optimization: *Quantum relaxations, quantum random access codes, rounding schemes*
+======================================================================================================================
+
+This material provides a deeper look into the concepts behind Quantum
+Random Access Optimization.
+
+Relaxations
+-----------
+
+Consider a binary optimization problem defined on binary variables
+:math:`m_i \in \\{-1,1\\}`. The choice of using :math:`\pm 1` variables
+instead of :math:`0/1` variables is not important, but will be
+convenient in terms of notation when we begin to re-cast this problem in
+terms of quantum observables. We will be primarily interested in
+`quadratic unconstrained binary optimization
+(QUBO) <https://en.wikipedia.org/wiki/Quadratic_unconstrained_binary_optimization>`__
+problems, although the ideas in this document can readily extend to
+problems with more than quadratic terms, and problems with non-binary or
+constrained variables can often be recast as a QUBO (though this
+conversion will incur some overhead).
+
+Within mathematical optimization,
+`relaxation <https://en.wikipedia.org/wiki/Relaxation_%28approximation%29>`__
+is the strategy of taking some hard problem and mapping it onto a
+similar version of that problem which is (usually) easier to solve. The
+core idea here is that for useful relaxations, the solution to the
+relaxed problem can give information about the original problem and
+allow one to heuristically find better solutions. An example of
+relaxation could be something as simple as taking a discrete
+optimization problem and allowing a solver to optimize the problem using
+continuous variables. Once a solution is obtained for the relaxed
+problem, the solver must find a strategy for extracting a discrete
+solution from the relaxed solution of continuous values. This process of
+mapping the relaxed solution back onto original problem’s set of
+admissible solutions is often referred to as **rounding**.
+
+For a concrete example of relaxation and rounding, see the
+`Goemans-Williamson Algorithm for
+MaxCut <https://en.wikipedia.org/wiki/Semidefinite_programming#Example_3_(Goemans%E2%80%93Williamson_max_cut_approximation_algorithm)>`__.
+
+Without loss of generality, the rest of this document will consider a
+`MaxCut <https://en.wikipedia.org/wiki/Maximum_cut>`__ objective
+function defined on a graph :math:`G = (V,E)`. Our goal is to find a
+partitioning of our vertices :math:`V` into two sets (:math:`+1` and
+:math:`-1`), such that we maximize the number of edges which connect
+both sets. More concretely, each :math:`v_i \in V` will be assigned a
+binary variable :math:`m_i \in \\{0,1\\}`, and we will define the *cut*
```

Comment:
Brace `{...}` is not displayed.
![image](https://github.com/qiskit-community/qiskit-optimization/assets/31178928/935fbf2e-17dd-419d-b860-c0c08d4004cc)

Reply:
Btw, I think m_i should be in {-1, 1} not {0, 1}.